compiler/rustc_interface/src/tests.rs

```diff
@@ -782,6 +783,7 @@ fn test_unstable_options_tracking_hash() {
     tracked!(flatten_format_args, false);
     tracked!(force_unstable_if_unmarked, true);
     tracked!(fuel, Some(("abc".to_string(), 99)));
+    tracked!(function_return, FunctionReturn::default());
```

Comment:
this is supposed to use a non default value

Reply:
Good catch, done.